starwars/src/components/CardComponents/CardContainer.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Card from "./Card"
+import './Card.css';
+
+ function CardContainer(props) {
+  return(
+      <ul>
+          {props.charactersBlock.map(char => <Card char={char} name={char.name} gender={char.gender} height={char.height} hair={char.hair_color} />)} 
```

Comment:
Here, you have some unused props. You are passing in the whole char object into your `char={char}` prop, and in your Card component you are rendering `props.char.name, props.char.gender, etc`. So all of these other props, `name={char.name}, gender={char.gender}, etc` aren't being used. I would say use `char={char}` and remove the other props.